blog/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path, include
```

Comment:
Add blank line between different import groups
Use absolute import instead of relative